docs/source/using-discovery/workingwithgpu.rst

```diff
@@ -139,78 +181,127 @@ There are several versions of CUDA Toolkits on Discovery, as listed below.::
   cuda/11.2
   cuda/11.3
   cuda/11.4
+  cuda/11.7
 
-You can always use the ``module avail`` command to check for the latest software versions on Discovery as well. To see details on a specific CUDA toolkit version, use ``module show``. For example, ``module show cuda/11.4``.
+Use the ``module avail`` command to check for the latest software
+versions on Discovery. To see details on a specific CUDA toolkit
+version, use ``module show``. For example, ``module show cuda/11.4``.
 
-To add CUDA to your path use ``module load``. For example, type ``module load cuda/11.4`` to load version 11.4 to your path.
+To add CUDA to your path, use ``module load``. For example, type
+``module load cuda/11.4`` to load version 11.4 to your path.
 
-Use the command ``nvidia-smi`` (NVIDIA System Management Interface) inside a GPU node to get the CUDA driver information and monitor the GPU device.
+Use the command ``nvidia-smi`` (NVIDIA System Management Interface)
+inside a GPU node to get the CUDA driver information and monitor the
+GPU device.
 
 Using GPUs with PyTorch
 ========================
-You should use PyTorch with a conda virtual environment if you need to run the environment on the Nvidia GPUs on Discovery.
+You should use PyTorch with a conda virtual environment if you need to
+run the environment on the Nvidia GPUs on Discovery. The following
+example demonstrates how to build PyTorch inside a conda virtual
+environment for CUDA version 11.7.  
 
-The following examples demonstrate how to build PyTorch inside a conda virtual environment for CUDA version 11.7. 
-Make sure that you are on a GPU node before loading the environment and also please note that the installation does not work on k40m or k80 GPU's
-
-.. note::
- Note that you can reuse the tensorflow environment if you've already created one, no need to create a new one with the exact same setup
-
-PyTorch installation steps (with Anaconda libraries)::
-
-  srun --partition=gpu --nodes=1 --pty --gres=gpu:v100-sxm2:1 --ntasks=1 --mem=4GB --time=01:00:00 /bin/bash
-  module load cuda/11.7
-  module load anaconda3/2022.05
+.. note:: 
+   Make sure to be on a GPU node before loading the
+   environment. Additionally, the latest version of PyTorch is not
+   compatible with GPUs with CUDA version 11.7 or less. Hence, the
+   installation does not work on k40m or k80 GPU's. In order to see
+   what ``non-Kepler`` GPUs might be available, one can execute this
+   command::
+
+     sinfo -p gpu --Format=nodes,cpus,memory,features,statecompact,nodelist,gres
```

Comment:
I like providing this command since the users can check and see what is available. Should we include -t idle to only show the idle (+ reservation and drain states) or should it be simplified a little bit to:
sinfo -p gpu -t idle --Format=nodes,statecompact,gres
since we are only really looking for the gpus and state? I am not sure which is preferred but I was trying to get the full command to be slightly better formatted because the information was clipping for me and the %XX only works with the --format option.

Reply:
@joseph-cooke , what do you think about the following instead? 

sinfo -p gpu -t idle --Format=nodes,statecompact,cpus,memory,nodelist,gres

I do like to tell the user the node names, memory, & cpus associated with it.